test/integration/tests/abrmd_extended-sessions.sh

```diff
@@ -132,4 +132,33 @@ tpm2 sessionconfig session.ctx --enable-encrypt --enable-decrypt
 tpm2 getrandom 8 -S session.ctx
 tpm2 flushcontext session.ctx
 
+# test that name verification works
+tpm2 evictcontrol -c prim.ctx 0x81000000
+tpm2 readpublic -c prim.ctx -n name.bin
+tpm2 startauthsession -S session.ctx --policy-session \
+  --tpmkey-context 0x81000000 -n name.bin
+# verify -c works
+tpm2 startauthsession -S session.ctx --policy-session \
+  -c 0x81000000 -n name.bin
+
+# test if name is bad, generate a random name of of the same size
+size=$(stat --printf="%s" name.bin)
```

Comment:
@williamcroberts, I am not sure why cirrus-ci doesn't like what appears to be a valid command line option for the stat tool.
```
stat: illegal option -- -
usage: stat [-FLnq] [-f format | -l | -r | -s | -x] [-t timefmt] [file|handle ...]
size=$(stat --printf="%s" name.bin) on line 145 failed: 1
```

Reply:
> @williamcroberts, I am not sure why cirrus-ci doesn't like what appears to be a valid command line option for the stat tool.
> 
> ```
> stat: illegal option -- -
> usage: stat [-FLnq] [-f format | -l | -r | -s | -x] [-t timefmt] [file|handle ...]
> size=$(stat --printf="%s" name.bin) on line 145 failed: 1
> ```

Ha yeah, bsd doesn't have all the gnu extensions on tools.